webpack.config.ts

```diff
@@ -184,8 +185,10 @@ module.exports = {
       inject: false
     }),
     new HtmlWebpackPlugin({
-      chunks: ['get_started'],
+      chunks: ['app', 'get_started'],
       filename: 'get_started.html',
+      meta: Utils.getMetas().getStarted,
+      loader: 'raw-loader',
```

Comment:
Why is this here?

Reply:
Oops
I was testing a different loader then
Will remove